index.js

```diff
@@ -31,11 +31,11 @@ async function handleRequest(request) {
       return Response.json(
         {
           code: '405 Method Not Allowed',
-          message: 'POST not valid for individual keys. Did you mean PUT?'
+          message: 'POST not valid for individual keys. Did you mean PUT?',
         },
         {
-          status: 405
-        }
+          status: 405,
+        },
```

Comment:
Just curious, were these commas added by a linter or was it something you did? I don't mind them but its funny they got added everywhere

Reply:
VS Code being VS Code

Reply:
lol